rst/indigo/release-notes/indigo-1.10.0.md

```diff
@@ -0,0 +1,19 @@
+# Indigo 1.10.0
+
+*22 March 2023*
+
+## Features
+* #941 CDX export
+
+## Bugfixes
+* #1003 Some texts are not rendered and may lead to Indigo crash
```

Comment:
fix ticket number point to indigo github
[#1003](https://github.com/epam/Indigo/issues/1003)

Reply:
fixed